.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: test
+on: [push, pull_request]
+jobs:
+  test:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: php-5.6
+            os: ubuntu-18.04
+            php-version: '5.6'
+          - name: php-7.0
+            os: ubuntu-18.04
+            php-version: '7.0'
+          - name: php-7.1
+            os: ubuntu-18.04
+            php-version: '7.1'
+          - name: ubuntu-18
+            os: ubuntu-18.04
+            php-version: '7.2'
+          - name: ubuntu-20
+            os: ubuntu-20.04
+            php-version: '7.4'
+# phpunit 5 and 6 require php < 8
```

Comment:
Why adding this comment on this YAML file?

Reply:
It explains that I could not test on php version >= 8 because the test file required phpunit 5 or 6, and those versions of phpunit require php version < 8. I'll try to remove phpunit instead.

Reply:
Done, tests are now successful on PHP 8.1.0 too.